src/media_changer.py

```diff
@@ -3924,8 +3924,9 @@ def do_retry(self, ret_val, returned_by_mtx_mount=False):
                         update_db = True
                     else:
                         if 'mtx: Request Sense: Additional Sense Code = 3B' in ret_val[3]:
-                            if 'mtx: Request Sense: Additional Sense Qualifier = 0D' in ret_val[3]:
```

Comment:
Frankly I do not understand this code 
```
if "a" in ret_val[3]:
   if "b" in  ret_val[3]:
```
what is ret_val[3] ? is that a list like ("a", "b", ...)?
I assume this is , but just wanted to ask is it?

Reply:
Yes, AIUI ret_val is the tuple in this error message:
```
[enstore@-- ~]$ enstore med --dismount VOL 278 LIB
BAD STATUS (1, 1, '', ['', 'mtx: Request Sense: Long Report=yes', 'mtx: Request Sense: Valid Residual=no', 'mtx: Request Sense: Error Code=70 (Current)', 'mtx: Request Sense: Sense Key=Illegal Request', 'mtx: Request Sense: FileMark=no', 'mtx: Request Sense: EOM=no', 'mtx: Request Sense: ILI=no', 'mtx: Request Sense: Additional Sense Code = 3B', 'mtx: Request Sense: Additional Sense Qualifier = 11', 'mtx: Request Sense: BPV=no', 'mtx: Request Sense: Error in CDB=no', 'mtx: Request Sense: SKSV=no', 'MOVE MEDIUM from Element Address 278 to 6920 Failed', ''])```

So, ret_val[3] is a list of.. error strings returned by SCSI, as parsed by mtx, I think.

Reply:
I've tried to change the error handling implementation as little as possible